src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write code here
+
+const items = document.querySelector('li');
+const tree = document.querySelector('.tree');
+
+for (const item of items) {
+  const span = document.querySelector('span');
+
+  item.prepend(span);
+  span.append(span.nextSibling);
+
+  tree.onclick = (e) => {
+    if (e.target.tagName !== 'SPAN') {
+      return;
+    }
+
+    const childrenContainer = e.target.parentNode.querySelector('ul');
```

Comment:
The code assumes that `childrenContainer` will always be found. Consider adding a check to ensure `childrenContainer` is not null before attempting to toggle its `hidden` property.